draft-ietf-core-new-block.xml

```diff
@@ -1134,20 +1169,20 @@ X<--: Message loss (response)
          +--->X     | NON PUT /path M:0x1b T:0xeb RT=11 QB1:10/1/1024
          +--------->| NON PUT /path M:0x1c T:0xec RT=11 QB1:11/1/1024
 ]]></artwork>
-          </figure></t>
-
-        <t>On seeing a payload from the next set of payloads, the server
-        realizes that some blocks are missing from the previous MAX_PAYLOADS
-        payloads and asks for the missing blocks in one go (<xref
-        target="B3non2"></xref>). It does so by indicating which blocks have
-        been received in the data portion of the response. The token used in
-        the response should be the token that was used in the last block
-        number received payload. The client can then derive the Request-Tag by
-        matching the token with the sent request.</t>
-
-        <t><figure anchor="B3non2"
-            title="Example of NON Request with Q-Block1 Option (Blocks Recovery)">
-            <artwork><![CDATA[        CoAP        CoAP
+            </figure></t>
+
+          <t>On seeing a payload from the next set of payloads, the server
+          realizes that some blocks are missing from the previous MAX_PAYLOADS
+          payloads and asks for the missing blocks in one go (<xref
+          target="B3non2"></xref>). It does so by indicating which blocks have
```

Comment:
Why are we deviating from reporting missing block to ACking received ones?

Reply:
Well caught - now have `blocks not been received` - fixed in 2 places.